fe/src/main/java/org/apache/doris/analysis/AdminRepairTableStmt.java

```diff
@@ -62,8 +63,12 @@ public void analyze(Analyzer analyzer) throws UserException {
 
         tblRef.getName().setDb(dbName);
 
-        if (tblRef.getPartitions() != null && !tblRef.getPartitions().isEmpty()) {
-            partitions.addAll(tblRef.getPartitions());
+        PartitionNames partitionNames = tblRef.getPartitionNames();
```

Comment:
Could we encapsulate the following logic?
```
 PartitionNames partitionNames = tblRef.getPartitionNames();
 if (partitionNames != null) {
            if (partitionNames.isTemp()) {
                throw new AnalysisException("xxxx");
            }
```

Reply:
I can just change it to:
```
if (tblRef.isTempPartitionSpecified()) {
        throw new AnalysisException("Do not support (cancel)repair temporary partitions");
}
if (tblRef.getPartitionNames() != null) {
        partitions.addAll(tblRef.getPartitionNames().getPartitionNames());
}
```

Does it better?

Reply:
I don't think which is better.  Let's keep it unchanged.